source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/ray_caster/patterns/patterns_cfg.py

```diff
@@ -122,3 +122,22 @@ class BpearlPatternCfg(PatternBaseCfg):
         We manually set the vertical ray angles to match the Bpearl sensor. The ray-angles
         are not evenly spaced.
     """
+
+
+@configclass
+class LidarPatternCfg(PatternBaseCfg):
+    """Configuration for the LiDAR pattern for ray-casting."""
+
+    func: Callable = patterns.lidar_pattern
+
+    channels: int = MISSING
+    """Number of Channels (Beams). Determines the vertical resolution of the LiDAR sensor."""
+
+    vertical_fov_range: tuple[float, float] = MISSING
+    """Vertical field of view range in degrees."""
```

Comment:
NIT: Can we stick to radians for consistency with SI units everywhere? :)

Reply:
Bpearl pattern cfg is also in degrees, wanted to stay consistent with that one. Also the datasheets was in degrees.